tests/utils/config_utils_test.py

```diff
@@ -0,0 +1,50 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mava.utils.config_utils import flatten_dict
+
+test_mock_nested_dict = {"a": 1, "c": {"a": 2, "b": {"x": 5, "y": 10}}, "d": [1, 2, 3]}
```

Comment:
@RuanJohn, I think we can make this a pytest fixture, then remove the test class (just have the test functions) and feed the fixture as input to each test function. This isn't necessarily better, but it will be more consistent with the other testing code. :)